src/data-access/repositories/mongo/mongo-device-repository.ts

```diff
@@ -1,24 +1,56 @@
+/* eslint-disable @typescript-eslint/no-unused-vars */
 import { BaseMongoRepository } from './base-mongo-repository'
 import { DeviceRepository } from '../interfaces/device-repository'
 import { Device } from '../../../model/entities/device'
 import { MongoDatabase } from './mongo-database'
-import { MongoDevice, MongoEntityConverter } from './mongo-entity-converter'
+import { INewsDevice } from '../../../model/entities/inews-device'
+import { TelemetricsDevice } from '../../../model/entities/telemetrics-device'
 
-const DEVICE_COLLECTION_NAME: string = 'peripheralDevices'
+const DEVICE_COLLECTION_NAME: string = 'externalDevices'
 
 export class MongoDeviceRepository extends BaseMongoRepository implements DeviceRepository {
```

Comment:
Your new methods: `findById`, `create` and `delete` is not defined on the interface `DeviceRepository`. You need to defined them there as well :)